src/main/java/Main.java

```diff
@@ -0,0 +1,172 @@
+            String name = request.queryParams("submitName");
+            String password = request.queryParams("submitPassword");
+            User user = users.get(name);
+
+            if(user == null) {
+
+                user = new User(name, password);
+                users.put(name, user);
+                Session session = request.session();
+                session.attribute(SESSION_USER, name);
+                response.redirect("/");
+            }
+            else if (user != null && request.queryParams("submitPassword").equals(user.getPassword())){
+                Session session = request.session();
+                session.attribute(SESSION_USER, name);
+                response.redirect("/");
+            }
+            else {
+                response.redirect("/login");
+            }
+            return "";
+        });
+
+        Spark.get("/create-bottle", (request, response) -> {
+            Map m = new HashMap();
+            return new ModelAndView(m, "index.html");
+        }, new MustacheTemplateEngine());
+
+        Spark.post("/create-bottle", ((request, response) -> {
+            Session session = request.session();
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+            if(user==null){
+                response.redirect("/");
+                return "";
+            }
+
+            String bottleName = request.queryParams("bottleName");
+            String bottleProducer = request.queryParams("bottleProducer");
+            String bottleRegion = request.queryParams("bottleRegion");
+            int bottleVintage = Integer.valueOf(request.queryParams("bottleVintage"));
+            String bottleVariety = request.queryParams("bottleVariety");
+            float bottleABV = Float.valueOf(request.queryParams("bottleABV"));
+            Bottle bottle = new Bottle(bottleName, bottleProducer, bottleRegion, bottleVintage, bottleVariety, bottleABV);
+            user.myBottles.add(bottle);
+            allBottles.add(bottle);
+            response.redirect("/");
+            return "";
+        }));
+
+        Spark.post("/delete-bottle", (request, response) -> {
+            Session session = request.session();
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+            String bottleName = request.queryParams("bottleName");
+
+            Bottle foundBottle = null;
+            for(Bottle bottle : user.myBottles){
+                if(bottle.getName().equals(bottleName)){
+                foundBottle = bottle;
+                }
+                user.myBottles.remove(foundBottle);
+                allBottles.remove(foundBottle);
+            }
+            response.redirect("/");
+            return "";
+        });
+        Spark.get("/edit-bottle", (request, response) -> {
+            Map m = new HashMap();
+            return new ModelAndView(m, "bottles.html");
```

Comment:
This can be condensed to one line `return new ModelAndView(new HashMap(), "bottles.html");`